services/app/handlers/addToWatchlist.js

```diff
@@ -0,0 +1,17 @@
+const firebase = require("../utils/firebase");
```

Comment:
inside `/services` TypeScript was not used to save time on the setup of the parser, as when zipping the code would require to be transpiled to `.js` before sending to AWS